libexec/slurm/monitor.sh

```diff
@@ -119,13 +128,17 @@ fi
 if [[ "$start_time" == "Unknown" ]] ; then
   start_time=""
 fi
+if [[ "$estimated_start_time" == "Unknown" ]] ; then
+  estimated_start_time=""
+fi
 if [[ "$end_time" == "Unknown" ]] ; then
   end_time=""
 fi
 
 # Render and return the payload
 echo '{}' | jq  --arg state "$state" \
                 --arg reason "$reason" \
+                --arg estimated_start_time "$estimated_start_time" \
```

Comment:
Do we not need a similar line for the estimated end time?

Reply:
Nope, that looks to be correct. See commit:

2403ddace18ef96a2b914a880355c18cad2d8ac8

Reply:
5b98b81d85392e49a356e42a305e1e98ed90db59